WebApiCoreSeed.WebApi/Controllers/ValuesController.cs

```diff
@@ -1,45 +1,53 @@
-﻿using System;
+
+namespace WebApiCoreSeed.WebApi.Controllers
+{
+
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Authorization;
 
-namespace WebApiCoreSeed.WebApi.Controllers
-{
+
     [Route("api/[controller]")]
     public class ValuesController : Controller
     {
         // GET api/values
         [HttpGet]
         public IEnumerable<string> Get()
         {
+            //TODO: Add Functionality GetData from database

```

Comment:
Minor grammar correction: Instead of "Add Functionality GetData" it should be "Add GetData functionality".

It also applies to the rest of the comments in this file.